shard.go

```diff
@@ -117,6 +117,16 @@ func (s *cacheShard) getValidWrapEntry(key string, hashedKey uint64) ([]byte, er
 	return wrappedEntry, nil
 }
 
+func (s *cacheShard) contains(key string, hashedKey uint64) bool {
+	s.lock.RLock()
+	defer s.lock.RUnlock()
```

Comment:
defer might have negative impact so we decided to not use it:
- https://github.com/allegro/bigcache/pull/37
- https://github.com/allegro/bigcache/pull/46
although things has changed so maybe it will not have a negative impact in general but in this case we can release lock before reading key from entry

Reply:
Done.

I thought the code was not using `defer` because it was made before the go 1.14 optimisations on defer statements, but the code is simple enough to not rely on it :-)